backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/calendar/BpkCalendarController.kt

```diff
@@ -75,7 +75,7 @@ class BpkCalendarController(
      * Updates the parameters of the calendar.
      */
     fun setParams(value: CalendarParams) {
-        _state = _state.copy(params = value)
+        _state = CalendarState(value, _state.selection)
```

Comment:
assigning to new CalendarState to update Cells as well, as currently the state is updating the params but don't reflect the changes on the current cells
```
@Immutable
data class CalendarState(
    val params: CalendarParams,
    val selection: CalendarSelection = CalendarSelection.None,
    @InternalBackpackApi val cells: CalendarCells = CalendarCells(params, selection),
)

```